src/metabase/models/setting.clj

```diff
@@ -323,9 +336,10 @@
   (or (not *enforce-setting-access-checks*)
       (nil? api/*current-user-id*)
       api/*is-superuser?*
-      (and
-       (allows-user-local-values? setting)
-       (not= (:visibility setting) :admin))))
+      (or (has-advanced-setting-access?)
```

Comment:
why does this need to be in another nested `or`? can't it just be at the top level?